instance/tasks.py

```diff
@@ -47,6 +49,9 @@
 
 logger = logging.getLogger(__name__)
 
+# Constants  ##################################################################
+
+KILL_ZOMBIES_CRON_SCHEDULE = getattr(settings, "KILL_ZOMBIES_SCHEDULE").split()
```

Comment:
@spokerman12, is there a reason for using `getattr()` instead of just `settings.KILL_ZOMBIES_SCHEDULE`, which is defined in the Django settings and has a default value defined.

Reply:
The first implementation used "environment variables" as it was discussed.